api/app/signals/apps/api/v1/validation/address/pdok.py

```diff
@@ -40,7 +40,10 @@ def _pdok_request_query_params(self, address, lon=None, lat=None):
             query_dict.update({'fq': f'woonplaatsnaam:{address["woonplaats"]}'})
         if 'postcode' in address and address["postcode"]:
             query_dict.update({'fq': f'postcode:{address["postcode"]}'})
-        query_dict.update({'fq': f'gemeentenaam:{",".join(DEFAULT_PDOK_MUNICIPALITIES)}'})
+
+        # remove None, '', ' ' strings before formatting
```

Comment:
Remove 'None' from this comment. Having `None` in `DEFAULT_PDOK_MUNICIPALITIES` will get you a `TypeError`.

Reply:
fixed ...